src/components/Map/RSVMap.tsx

```diff
@@ -34,16 +35,24 @@ export const RSVMap: React.VFC<Props> = ({ geometry }) => {
   };
   const [consent, setConsent] = useState<boolean | null>(null);
   useEffect(() => setConsent(getOptInCookie()));
-  if (consent === null) {
-    return null;
-  }
-  if (consent === false) {
-    return <OptIn setConsent={setConsent} />;
+  if (!consent) {
+    return (
+      <div className="relative">
+        {consent === false && (
+          <div className="absolute top-1/2 z-10 mx-2 -translate-y-1/2">
+            <OptIn setConsent={setConsent} />
+          </div>
+        )}
+        <StaticImage
+          src="./map.png"
+          alt="Accept the Privacy Policy to view the map"
+        />
```

Comment:
Apart from that the layout doesn't seem to be right, maybe we could apply a gaussian blur, so it's clearer, that the map the consent dialog is floating "above"   it?

E![Bildschirmfoto von 2022-04-13 09-09-17](https://user-images.githubusercontent.com/6567206/163120182-d66f60c1-8c37-4456-914a-ea91f0841670.png)